test/scalapack_wrappers.hh

```diff
@@ -3714,22 +3787,23 @@ inline void scalapack_pstedc(
 //------------------------------------------------------------------------------
 // Templated wrapper
 template <typename scalar_t>
-void scalapack_pstedc(
-    const char* jobz, int64_t n,
+void stedc(
+    lapack::Job jobz, int64_t n,
     scalar_t* D, scalar_t* E,
     scalar_t* Z, int64_t iz, int64_t jz, blas_int* descZ,
     scalar_t* work, int64_t lwork,
     blas_int* iwork, int64_t liwork,
     int64_t* info )
 {
+    const char* jobz_ = to_c_string( jobz );
```

Comment:
LAPACK++ uses `char compz_ = to_char_comp( compz );` for `stedc`. ScaLAPACK has `compz` instead of `jobz`, so fix this.